pkg/workflows/ring/pb/consensus.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package ring;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/workflows/ring/pb";
+
+message Observation {
+  map<uint32, bool> shardHealthStatus = 1; // shard_id -> is_healthy
+  repeated string workflowIDs = 2;
+  google.protobuf.Timestamp now = 3;
```

Comment:
I'm assuming this is used for expiry, see my comment about not using expiration.